### PR TITLE
completed task 3.5

### PR DESCRIPTION
**Pull request by DanielCouperthwaite**
Hi,

I think I have completed task 3.5, please check when you get a chance and let me know if I need to change anything. 

Thanks!

### CODE REVIEW

File: controllers/controllers.js [hunk 1]
```diff
@@ -10,3 +11,11 @@ exports.getTopics = (req, res) => {
       });
 }
 
+exports.getApiInfo = (req, res, next) => {
+    console.log('in controller')
```
Comment (B7Bprog) [inline controllers/controllers.js:15]: Please remove all logs from the finished code!

File: models/models.js [hunk 2]
```diff
@@ -1,8 +1,17 @@
 const connection = require('../db/connection')
+const fs = require('fs/promises')
+// const endpoints = require('../endpoints.json')
 
 exports.fetchTopics = () => {
     console.log('in model')
```
Comment (B7Bprog) [inline models/models.js]: Log to be removed.

File: models/models.js [hunk 3]
```diff
@@ -1,8 +1,17 @@
 const connection = require('../db/connection')
+const fs = require('fs/promises')
+// const endpoints = require('../endpoints.json')
```
Comment (B7Bprog) [inline models/models.js]: Remove commented out code.

File: models/models.js [hunk 4]
```diff
@@ -1,8 +1,17 @@
...
 exports.fetchTopics = () => {
     console.log('in model')
     return connection.query('SELECT * FROM topics;').then((result) => {    
         return result.rows;
     })
-}
\ No newline at end of file
+}
+
+// exports.fetchApiInfo = () => {
```
Comment (B7Bprog) [inline models/models.js]: All commented out code should be removed.

File: models/models.js [hunk 5]
```diff
@@ -1,8 +1,17 @@
 const connection = require('../db/connection')
+const fs = require('fs/promises')
```
Comment (B7Bprog) [inline models/models.js]: This is not used.

**Review by B7Bprog — APPROVED** [review]
Hey Daniel, please see my comments and do those minor changes, then it's good to merge. I see you've tried to use 'fs' to read the file, which is a good idea. If you want, you can try to make it work to make sure we always reply with 'fresh' data, but I leave that up to you. Otherwise feel free to merge. 👍

File: controllers/controllers.js (hunk 1 above)
Comment (DanielCouperthwaite) [reply controllers/controllers.js:15]: That's perfect, thanks! I forgot all about the console.logs as soon as I sent them so thank you for the reminder! I'll remove them, and all in future before pushing.

**Pull request merged**